static/app/views/replays/detail/focusArea.tsx

```diff
@@ -32,18 +33,22 @@ function FocusArea({replay}: Props) {
     useReplayContext();
   const organization = useOrganization();
 
-  const event = replay.getEvent();
-  const spansEntry = replay.getEntryType(EntryType.SPANS);
-
   // Memoize this because re-renders will interfere with the mouse state of the
   // chart (e.g. on mouse over and out)
   const memorySpans = useMemo(() => {
-    return replay.getRawSpans().filter(replay.isMemorySpan);
+    return replay?.getRawSpans().filter(replay.isMemorySpan);
   }, [replay]);
 
+  if (!replay || !memorySpans) {
+    return <Placeholder height="150px" />;
+  }
```

Comment:
If this happens before the `const memorySpans = ` statement, could we drop the `|| !memorySpans` part?

Reply:
eslint will yell because of conditional hooks